src/components/Results.tsx

```diff
@@ -1,31 +1,58 @@
-import { Grid } from '@mui/material';
+import { Box, Container, Grid, Pagination } from '@mui/material';
 import ResultCard from './ResultCard';
 import { Result, ResultList } from '../types/result';
+import { useState } from 'react';
 
-type ResultsProps = {
+interface ResultsProps {
   searchResults: ResultList;
-};
+}
 
 const Results = ({ searchResults }: ResultsProps) => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [itemsPerPage] = useState(10);
```

Comment:
`useState` hook returns a value variable and a setter function (exactly like `currentPage` and `setCurrentPage`) but here you only declared a value without a setter, and does this value of 10 need to be stored as state?

Reply:
yeah, no need for useState...